### PR TITLE
Add a solution for Homework 9 Chapter 1

### PR DESCRIPTION
**Pull request by tzurowski**

### CODE REVIEW

File: Src/BootCamp.Chapter/ArrowMovement.cs [hunk 1]
```diff
@@ -18,7 +18,22 @@ public static class ArrowMovement
         /// <returns>One of the arrow characters. '↥' by default.</returns>
         public static char GetIndicator(char symbol)
         {
-            return '-';
+            switch (symbol)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/ArrowMovement.cs]: Well done- this is clear and simple.

File: Src/BootCamp.Chapter/ArrowMovement.cs [hunk 2]
```diff
@@ -18,7 +18,22 @@ public static class ArrowMovement
...
+                case 'a':
+                    return '↤';
+                case 'S':
+                case 's':
+                    return '↧';
+                case 'D':
+                case 'd':
+                    return '↦';
+            }
+            return '↥';
```
Comment (Almantask) [inline Src/BootCamp.Chapter/ArrowMovement.cs]: this should go to ``default:`` inside the switch

File: Src/BootCamp.Chapter/BinaryConverter.cs [hunk 3]
```diff
@@ -8,12 +8,50 @@ public static class BinaryConverter
...
+                    return false;
+                }
+            }
+            return true;
         }
 
         public static string ToBinary(long number)
         {
-            return "";
+            Stack<int> myBinaryStack = new Stack<int>();
```
Comment (Almantask) [inline Src/BootCamp.Chapter/BinaryConverter.cs]: You don't need a stack here- you can simply append the number to string right away.

File: Src/BootCamp.Chapter/ArrowMovement.cs (hunk 2 above)
Comment (tzurowski) [reply Src/BootCamp.Chapter/ArrowMovement.cs]: You're right, changed.

File: Src/BootCamp.Chapter/BinaryConverter.cs (hunk 3 above)
Comment (tzurowski) [reply Src/BootCamp.Chapter/BinaryConverter.cs]: Changed, but my way can be little to mess :/

File: Src/BootCamp.Chapter/BinaryConverter.cs [hunk 4]
```diff
@@ -37,20 +37,22 @@ private static bool IsBinary(string binary)
 
         public static string ToBinary(long number)
         {
-            Stack<int> myBinaryStack = new Stack<int>();
+            string binaryWord = "";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/BinaryConverter.cs]: Prefer to use ``StringBuilder`` instead when concatinating a string in a loop. It is much faster (like a hundred times faster)

**Pull request closed without merging**